stackblitz-starter/package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This template is based on the [StackBlitz react-ts template](https://github.com/stackblitz/starters/tree/main/react-ts).

Reply:
yeah this is "faster" because it uses cached CRA template that is completely controlled by cloud provider (SB), no matter what changes we will made on that live edit they wont get reflected - one of many reasons why we moved away from codesanbox.